model/telemetry/metrics.yaml

```diff
@@ -0,0 +1,75 @@
+groups:
+  - id: metric.telemetry.sdk.trace.spans.ended
+    type: metric
+    metric_name: telemetry.sdk.trace.spans.ended
+    stability: experimental
+    brief: "The number of spans which have been ended"
+    instrument: counter
+    unit: "1"
+
+  - id: metric.telemetry.sdk.trace.spans.sampled
+    type: metric
+    metric_name: telemetry.sdk.trace.spans.sampled
+    stability: experimental
+    brief: "The number of spans which have been ended AND are sampled"
+    instrument: counter
+    unit: "1"
+
+  - id: metric.telemetry.sdk.trace.processor.queue_size
```

Comment:
Wouldn't the name with `span` instead of `trace` be more intuitive here?:
`metric.telemetry.sdk.span.processor.queue_size`

Since, in the description you refer to spans and the span processor.

... same for other related metrics

Reply:
also, the name `...queue_size` might be misleading here. May (and in alignment with the other related comments) something like this instead?:

`telemetry.sdk.span.processor.spans_queued`

Reply:
> also, the name ...queue_size might be misleading here. May (and in alignment with the other related comments) something like this instead?:

I feel like `spans_queued` sounds more like a counter of the number of spans which made it into the queue. In contrast, `queue_size` represents the number of spans in the queue at a given moment in time.
But not really a strong opinion here.